ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -176,7 +178,19 @@ final class Runner[T <: ReadWriteService, Extra](
                 _ <- participantConfig.mode match {
                   case ParticipantRunMode.Combined | ParticipantRunMode.LedgerApiServer =>
                     for {
+                      dbSupport <- DbSupport
+                        .owner(
+                          jdbcUrl = apiServerConfig.jdbcUrl,
+                          serverRole = ServerRole.ApiServer,
+                          connectionPoolSize = apiServerConfig.databaseConnectionPoolSize,
+                          connectionTimeout = apiServerConfig.databaseConnectionTimeout,
+                          metrics = metrics,
+                        )
+                        .acquire()
+                      userManagementStore =
+                        new InMemoryUserManagementStore // TODO persistence wiring comes here
```

Comment:
Is the idea that the user management store uses the `dbSupport` instance created above? That would mean the indexer and the user management store (part of the ledger api server) share the same connection pool. Could there be any issues with that? So far the indexer connection pool was isolated from all ledger api operations.

Reply:
Well they are sharing the same db. I was thinking having a different connection pool only for user management makes not much sense: no isolation needed, they are equally important, they need to be equally fast (endpoint auth might read from db). Keeping around a connection pool just for this would be probably quite a waste on the postgres side.

Reply:
Doh, I have confused Index and Indexer and thought you wanted to use the Indexer connection pool for the user management store. Using the ledger api server connection pool of course makes sense.